README.md

```diff
@@ -25,8 +26,11 @@ To disclose a security issue to our team, [please submit a report via HackerOne
 ## Support
 This repository is not suitable for support. Please don't use our issue tracker for support requests, but for core WooCommerce issues only. Support can take place through the appropriate channels:
 
-* The [WooCommerce premium support portal](https://woocommerce.com/my-account/create-a-ticket/) for customers who have purchased themes or extensions.
+* If you have a problem, you may want to start with the [self help guide](https://docs.woocommerce.com/document/woocommerce-self-service-guide/).
+* The [WooCommerce.com premium support portal](https://woocommerce.com/contact-us/ ) for customers who have purchased themes or extensions.
 * [Our community forum on wp.org](https://wordpress.org/support/plugin/woocommerce) which is available for all WooCommerce users.
+* The WooCommerce Help and Share Facebook group.
```

Comment:
Maybe add link (if there is one)

Reply:
I don't have a link to this group. This was already in the documentation and I just moved it here. We might want to reevaluate this list.